tests/FakeItEasy.Specs/ValueTaskReturnValueSpecs.cs

```diff
@@ -0,0 +1,166 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Xbehave;
+
+    public class ValueTaskReturnValueSpecs

```

Comment:
The `Task` equivalent was covered (at great length) in unit tests, but we tend to prefer specs now. At some point we might want to migrate the `Task` tests to specs.